WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
```

Comment:
For consistency with the rest of the website, I believe all the labels in this file should be actual `<label>...</label>` elements. This will also make things nicer for people when clicking around.

Also, I believe we should add class `form-control` to all these inputs. If you don't like how `form-control`s look, then I'm open to changing the default styling for them somewhere else, but for now, consistency is more important.